tap_mailchimp/schemas/list_segment_members.json

```diff
@@ -162,6 +163,38 @@
         "null",
         "string"
       ]
+    },
+    "last_note": {
+      "type": [
+        "object"
```

Comment:
Will this schema force `last_note` to be object?

Reply:
Added “null” as the fallback option.